modules/Navbar.js

```diff
@@ -0,0 +1,51 @@
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
+
+const formSection = document.getElementById('form-section');
+const formLink = document.getElementById('form-link');
+const homeLink = document.getElementById('home-link');
+const contactLink = document.getElementById('contact-link');
+const booksSection = document.getElementById('all-books');
+const titleSection = document.getElementById('titleSection');
+const contactSection = document.getElementById('contact-section');
```

Comment:
- [ ] When I click on the sections (list, contact ) they aren't working please make sure when you click on a section it should take you to it